ompi/mca/pml/ucx/pml_ucx.c

```diff
@@ -230,22 +233,37 @@ int mca_pml_ucx_open(void)
 
     /* Query UCX attributes */
     attr.field_mask        = UCP_ATTR_FIELD_REQUEST_SIZE;
+#if HAVE_UCP_ATTR_MEMORY_TYPES && OPAL_CUDA_SUPPORT
```

Comment:
no need for OPAL_CUDA_SUPPORT
use `#if HAVE_UCP_ATTR_MEMORY_TYPES`

Reply:
fixed.